test/config/group2/case3.yaml

```diff
@@ -0,0 +1,96 @@
+- method: SetNodeToBad
+  parameters:
+    nodeName: 0.0.0.4
+- method: SetNodeToBad
+  parameters:
+    nodeName: 0.0.0.5
+- method: SetNodeToBad
+  parameters:
+    nodeName: 0.0.0.6
+- method: SchedulePod
+  parameters:
+    podName: group1_pod1
+    phase: "Preempting"
+    podGroupSchedulingRequest:
+      version: v2
+      virtualCluster: VC1
+      priority: 1
+      pinnedCellId: ""
+      cellType: V100
+      cellNumber: 1
+      gangReleaseEnable: false
+      lazyPreemptionEnable: false
+      podRootGroup:
+        name: group1
+        withinOneCell: ""
+        pods:
+        - podMinNumber: 1
+          podMaxNumber: 1
+          cellsPerPod:
+            cellType: V100
+            cellNumber: 1
+          containsCurrentPod: true
+        childGroups: []
+- method: AssertPodBindResult
+  parameters:
+    podName: group1_pod1
+    expectedResult:
+      node: 0.0.0.1
```

Comment:
should be `0.0.0.3` here, because its sibling `0.0.0.4` is a bad cell, the cell allocation algorithm will choose the existing node level cell instead of splitting a higher rack level cell.

Reply:
done